.merlin

```diff
@@ -0,0 +1,3 @@
+PKG cmdliner cohttp cstruct lwt nbd re.str sha tapctl tar threads uri vhd-format xenstore xenstore_transport
```

Comment:
I think it would be better to have one package per line - this makes diffs easier.